chat/src/main/java/net/md_5/bungee/chat/TextComponentSerializer.java

```diff
@@ -32,13 +30,9 @@ public TextComponent deserialize(JsonElement json, Type typeOfT, JsonDeserializa
     @Override
     public JsonElement serialize(TextComponent src, Type typeOfSrc, JsonSerializationContext context)
     {
-        List<BaseComponent> extra = src.getExtra();
         JsonObject object = new JsonObject();
```

Comment:
What's the logic behind this change?

Reply:
It's consistent with the other concrete types of component serializers (none of them check if the base component has anything to serialize before invoking the serialize method for it. If there is nothing to serialize, the base component serializer already takes care of that).